cluster/gce/gci/master-helper.sh

```diff
@@ -142,9 +145,11 @@ function create-master-instance-internal() {
     enable_ip_aliases=false
   fi
 
-  local network=$(make-gcloud-network-argument \
+  # shellcheck disable=SC2153 # 'NETWORK' is assigned by upstream
+  network=$(make-gcloud-network-argument \
     "${NETWORK_PROJECT}" "${REGION}" "${NETWORK}" "${SUBNETWORK:-}" \
     "${address:-}" "${enable_ip_aliases:-}" "${IP_ALIAS_SIZE:-}")
+  local network
```

Comment:
I'd expect `local network` to come before the assignment to `$network`. Did I miss a detail?

Reply:
Thanks @sftim! fixed